packages/core/package.json

```diff
@@ -54,12 +49,12 @@
     "/config/*.json"
   ],
   "scripts": {
-    "test": "yarn mocha-test && yarn buidler-test",
-    "buidler-test": "node --max-old-space-size=1024 $(yarn bin buidler) test",
+    "test": "yarn mocha-test && yarn hardhat-test",
+    "hardhat-test": "node --max-old-space-size=1024 $(yarn bin hardhat) test",
     "mocha-test": "mocha mocha-test --recursive",
     "load-addresses": "yarn run apply-registry",
     "clean": "rm -rf build",
-    "build": "yarn truffle compile && yarn load-addresses",
+    "build": "yarn truffle compile && yarn load-addresses && yarn hardhat compile",
```

Comment:
this additional `yarn hardhat compile` might not be required but I just added it for completeness.